.env.example

```diff
@@ -46,8 +46,7 @@ PGSSLMODE=
 # @see: https://github.com/calendso/calendso/issues/263
 # @see: https://next-auth.js.org/configuration/options#nextauth_url
 # Required for Vercel hosting - set NEXTAUTH_URL to equal your NEXT_PUBLIC_WEBAPP_URL
-# NEXTAUTH_URL='http://localhost:3000'
-NEXTAUTH_URL=
+NEXTAUTH_URL='http://localhost:3000'
```

Comment:
There are other env variables set to the localhost example in this file so there is no reason this one shouldn't be set as well.

Reply:
@zomars

Reply:
Agreed